build/checks/crosslinks.py

```diff
@@ -0,0 +1,27 @@
+import test, sys, re, json
+
+regex = r"{name}(?!</a>)"
+
+artists = []
+
+with open("./www/artists.json") as artf:
+    data = json.loads(artf.read())
+    for artist in data:
+        artists.append(artist["name"])
+
+class Crosslinks(test.Test):
+    def check(self, source, text):
+        for artist in artists:
+            name = artist.lower().split(" ")
+            if name[0][0] + name[-1] in source: continue
```

Comment:
I want to confirm that I understand what this file is doing, since you requested I review this pull request. This particular line checks if an artist's first initial and last name is in the source, ensuring that it doesn't request crosslinks for artists within their own site, correct? Did we handle the issue dealt with by #373? If not, those pages will almost certainly have the very problem we're trying to avoid with this line. They're making up 13 of the current errors. Same deal with Emma Lake - it's not named in the same format because it's not actually a person, it's an...event? I think?

Reply:
Emma Lake Artists' Workshop has 13 errors as well, 10 of which are for the name "Emma Lake Artists' Workshop" being in the emmalake folder. Handling this will likely be harder than the other errors - we could potentially rename the folder to eworkshop, but that feels clunky, and is more likely than any other change to create dead links - Emma Lake is a very common cross-link target.

Reply:
I am going to fix this to use the short hand from artist.json

Reply:
artist.json currently does not include a shorthand variable. Do you want one added? If so, that should be done before fixing the script, which should in turn be done before approving this commit.

Reply:
Ah, I can just get it from the path in that case